dev/SapphireTest.php

```diff
@@ -670,29 +670,36 @@ public function assertDOSContains($matches, $dataObjectSet) {
 	 * @param mixed $dataObjectSet The {@link SS_List} to test.
```

Comment:
I'm reviewing this and it seems like a cosmetic change unrelated to ChangeSet publishing. In future, I think it would be better if a change like this was broken out as a separate commit with a message that explains why you've done it. E.g. "PSR-2 compliance", "Restructured code for clarity", or "Fixed bug XYZ".